lib/offsite_payments/integrations/moneybookers.rb

```diff
@@ -174,23 +214,32 @@ def secret
           @options[:credential2]
         end
 
-        # Acknowledge the transaction to MoneyBooker. This method has to be called after a new
-        # apc arrives. It will verify that all the information we received is correct and will return a
-        # ok or a fail. The secret (second credential) has to be provided in the parameter :credential2
-        # when instantiating the Notification object.
+        # Acknowledge the transaction to MoneyBooker.
+        # This method has to be called after a new apc arrives.
+        # It will verify that all the information we received is correct
+        # and will return a ok or a fail.
+        # The secret (second credential) has to be provided in the parameter
+        # :credential2 when instantiating the Notification object.
         #
         # Example:
         #
-        #   def ipn
-        #     notify = Moneybookers.notification(request.raw_post, :credential2 => 'secret')
+        #  def ipn
+        #    notify = Moneybookers.notification(
+        #      request.raw_post, :credential2 => 'secret'
+        #    )
         #
-        #     if notify.acknowledge
-        #       ... process order ... if notify.complete?
-        #     else
-        #       ... log possible hacking attempt ...
-        #     end
+        #    if notify.acknowledge
+        #      ... process order ... if notify.complete?
+        #    else
+        #      ... log possible hacking attempt ...
+        #    end
+        #  end
         def acknowledge(authcode = nil)
-          fields = [merchant_id, item_id, Digest::MD5.hexdigest(secret.to_s).upcase, merchant_amount, merchant_currency, status_code].join
+          fields = [
+            merchant_id, item_id, Digest::MD5.hexdigest(secret.to_s).upcase,
+            merchant_amount, merchant_currency, status_code
+          ].join
```

Comment:
thanks for cleaning up the code. Please keep the esthetic changes in a separate commit